node/api.go

```diff
@@ -342,3 +343,57 @@ func (s *publicWeb3API) ClientVersion() string {
 func (s *publicWeb3API) Sha3(input hexutil.Bytes) hexutil.Bytes {
 	return crypto.Keccak256(input)
 }
+
+type ExecutionPoolThreads struct {
+	HttpLimit uint64
+	WSLimit   uint64
+}
+
+type ExecutionPoolRequestTimeout struct {
+	HttpLimit time.Duration
+	WSLimit   time.Duration
+}
+
+func (api *privateAdminAPI) GetExecutionPoolThreads() *ExecutionPoolThreads {
+	httpLimit := api.node.http.httpConfig.threads
```

Comment:
if these values can be changed in runtime, then we should have either atomics or a mutex for them